azerothcore/azerothcore-wotlk
# fix(DB/Quest) : Fixed Deliver the Plans to An'telas

## Changes Proposed:
-  Added the text line to the NPC Magister Sylastor(16237) after completing the quest
-  Added the summon of the 2 night elves

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6970

## SOURCE:

https://www.youtube.com/watch?v=hx-6cwi0jt4
https://tbc.wowhead.com/quest=9166/deliver-the-plans-to-antelas#comments:id=116114

## Tests Performed:

- .quest add 9160
- .quest complete 9160
- .go c id 16231 and pick quest
- .quest complete 9163
- Pick new quest
- .go c id 16237 and after completing check the line and the summoning of the elves

## How to Test the Changes:


1 .quest add 9160
2 .quest complete 9160
3 .go c id 16231 and pick quest
4 .quest complete 9163
5 Pick new quest
6 .go c id 16237 and after completing check the line and the summoning of the elves

Test video 

https://streamable.com/tvkpu6


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
